packages/scripts/README.md

```diff
@@ -54,6 +54,12 @@ This is how you execute the script with presented setup:
 
 * `npm run build` - builds the code for production.
 
+_Options_:
+
+- `--WP_BUNDLE_ANALYZER` : This flag is used to enables utility that represents bundle content as convenient interactive zoomable treemap.
+- `--WP_LIVE_RELOAD_PORT` : When present, the port on which live reload works when running watch mode will be changed. An example usage would be `npx wp-scripts build --WP_LIVE_RELOAD_PORT=1234`.
+- `--WP_DEVTOOL` : A development tool is designated to control how source maps are generated. More information can be found [here](https://webpack.js.org/configuration/devtool/#devtool).
```

Comment:
Both `WP_DEVTOOL` and `WP_LIVE_RELOAD_PORT` get only applied in development mode - this means it works only with `start` script.